org.icculus.tennix.yaml

```diff
@@ -15,23 +15,6 @@ finish-args:
   - --device=dri
 
 modules:
-  - shared-modules/glu/glu-9.json
-
-  - name: sdl12-compat
-    buildsystem: cmake-ninja
-    cleanup:
-      - '*.a'
-      - /include
-    sources:
-      - type: archive
-        url: https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz
-        sha256: 5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014
```

Comment:
This needs SDL2_gfx as a module, see https://www.ferzkopp.net/wordpress/2016/01/02/sdl_gfx-sdl2_gfx/